src/Framework/TestCase.php

```diff
@@ -1411,6 +1430,10 @@ protected function createPartialMock($originalClassName, array $methods): MockOb
      * Returns a test proxy for the specified class.
      *
      * @throws Exception
+     *
+     * @psalm-template RealInstanceType of object
+     * @psalm-param class-string<RealInstanceType>|string[] $originalClassName
```

Comment:
Parameter $originalClassName has wrong type 'array<array-key, string>|class-string<RealInstanceType>', should be 'string'
```
     * @psalm-param class-string<RealInstanceType>|string[] $originalClassName
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Remove `|string[]`